tile-generation/src/main/scala/com/oculusinfo/tilegen/pipeline/PipelineOperations.scala

```diff
@@ -269,6 +283,31 @@ object PipelineOperations {
 	}
 
 	/**
+	 * Convert the data type of a column
+	 *
+	 * @param sourceColSpec nae of the colum to be converted
+	 * @param columnFcn function doing the type conversion
+	 * @param columnType target data type
+	 * @param input PipelineData from previous stage
+	 * @return PipelineData containing updated Dataframe
+	 */
+	def convertColumnTypeOp (sourceColSpec: String, columnFcn: Array[Any] => Any, columnType: DataType)(input: PipelineData) = {
+
+		// Make sure the temp column name is unique
+		val tempCol = "temp-column-" + System.currentTimeMillis()
+
+		val columnAdded = addColumn(input.srdd, tempCol, columnType, c => columnFcn(c) , sourceColSpec)
+
+		val result = columnAdded.select(columnAdded.columns
+			.filter(_ != sourceColSpec)
+			.map(colName => new Column(colName))
+			.toList : _ *
```

Comment:
not a big deal, but is the toList necessary? It's an array already, which should work with :_*, shouldn't it?